admin_manual/configuration_server/reverse_proxy_configuration.rst

```diff
@@ -167,7 +167,7 @@ you can set the following parameters inside the :file:`config/config.php`.
     'overwriteprotocol' => 'https',
     'overwritewebroot'  => '/domain.tld/nextcloud',
     'overwritecondaddr' => '^10\.0\.0\.1$',
-    'overwrite.cli.url' => 'https://domain.tld/,
+    'overwrite.cli.url' => 'https://ssl-proxy.tld/domain.tld/',
```

Comment:
```suggestion
    'overwrite.cli.url' => 'https://ssl-proxy.tld/domain.tld/nextcloud',
```

Reply:
I wonder how relevant the above example block is nowadays. 
The time that someone would actually use a ssl proxy because the certificates are too expensive is long over.
We should probably remove the section because people confuse it with reverse proxies used today.

Reply:
See my more elaborated answer: https://github.com/nextcloud/documentation/issues/9466#issuecomment-1345339655